integration/spark/src/main/scala/org/apache/spark/sql/secondaryindex/util/SecondaryIndexUtil.scala

```diff
@@ -542,100 +542,6 @@ object SecondaryIndexUtil {
     indexToFactColMapping
   }
 
-  /**
-   * Identifies all segments which can be merged for compaction type - CUSTOM.
-   *
-   * @param sparkSession
-   * @param tableName
-   * @param dbName
-   * @param customSegments
-   * @return list of LoadMetadataDetails
-   * @throws UnsupportedOperationException   if customSegments is null or empty
-   */
-  def identifySegmentsToBeMergedCustom(sparkSession: SparkSession,
```

Comment:
Please revert this file changes. These are API's exposed to user for test purpose

Reply:
done